### ISSUE TITLE
Support for PostgresDB

### ISSUE THREAD

**Issue by Augigauki** [thgh/payload-plugin-oauth#16]
Hi! Not really a issue, but I didn't find another place to raise awareness for this.

I am working on a Payload 2.0^ project with a PostgresDB, and would love to use this plugin for enabling auth via Azure AD. During my testing now it seems as if this plugin requires a MongoDB-database. Would love it if it could also support postgres!

**Comment by thgh** [comment]
Agree! I think it's because passport needs sessions so I added express-session which needs some storage.

**Comment by Augigauki** [comment]
Do you think adding postgres-support is feasible? Got a project already running on postgres that I would love enabling azure-ad for, having to migrate to mongodb for it would be quite a big painpoint for sure. If I know a update is in the works enabling postgres-support I can just sit tightly and wait :)

**Comment by alsherif-khalaf** [comment]
> Do you think adding postgres-support is feasible? Got a project already running on postgres that I would love enabling azure-ad for, having to migrate to mongodb for it would be quite a big painpoint for sure. If I know a update is in the works enabling postgres-support I can just sit tightly and wait :)

Did you find a solution for this issue?

**Comment by bertribe** [comment]
I would like to see Postgres support as well
